core/pom.xml

```diff
@@ -222,6 +222,18 @@
             <version>1.1.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.littleshoot</groupId>
+            <artifactId>littleproxy</artifactId>
+            <version>1.0.0-beta7</version>
```

Comment:
Even if we perform changes to the PluginManager class, it seems to me that this change would be easy to re-apply, so +1 to move PluginManager to its own module first.